### ISSUE TITLE
Home Assistant Beta 0.109.0b0 Translations Error

### ISSUE THREAD

**Issue by mynameisdaniel32** [postlund/pyatv#617]
**Describe the bug**
Everything works, but getting a lot of cases of this in the logs:
```
2020-04-24 03:30:43 WARNING (MainThread) [homeassistant.helpers.translation] apple_tv: the '.translations' directory has been moved, the new name is 'translations', starting with Home Assistant 0.111 your translations will no longer load if you do not move/rename this 
```

**To Reproduce**
Home Assistant component loaded via HACS

**Expected behavior**

**System Setup (please complete the following information):**
 - OS: Debian
 - Python: 3.7
 - pyatv: unsure
- Apple TV: Apple TV 4K tvOS 13.4

**Additional context**

**Comment by postlund** [comment]
Noticed this as well. I'm planning on making an update next week and get pyatv 0.6.0 out the door. I will take care of this then 👍

**Comment by postlund** [comment]
Just pushed a new version that fixes this!

**Comment by mrardon** [comment]
> Just pushed a new version that fixes this!

I'm glad I found this issue, was pulling my hair out. After the integration broke this morning with 0.6.0. I uninstalled and reinstalled after 0.6.1 was released. I'm not yet on the beta of HA so I had no translations when adding the integration back. Apple TV in the integration add screen showed apple_tv and none of the dialog boxes showed any text. 

After finding this issue I copied the translations folder to .translations and it fixed it. So FYI for anyone in the future and may possibly wanna keep the .translations folder or make a note somewhere about it.
